### ISSUE TITLE
Travis CI improvements

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#293]
Can `py_requirements.txt` point pip to the version of `sphinx_rtd_theme` we are using? Talking with @matt-long, there are several possibilities based on [pip's ability to point to github](https://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source):

1. See if [the rtfd/sphinx_rtd_theme branch](https://github.com/rtfd/sphinx_rtd_theme/tree/version-dropdown) created for [#438](https://github.com/rtfd/sphinx_rtd_theme/pull/438) is the same as what we got from Unidata... if so, use that; if not, modify `conf.py` to be compatible with this branch
1. See if Unidata is interested in hosting their version of `sphinx_rtd_theme` in github rather than passing a `.whl` file around
1. Fork `rtfd/sphinx_rtd_theme` into `marbl-ecosys` organization, and either (a) use the `version-dropdown` branch mentioned in first option or (b) create a branch with the Unidata version of the package, then `py_requirements.txt` can point to our github organization.

I'm leaning towards option 1 or 3a -- it seems likely that if rtfd does eventually bring in support for a version dropdown, it will be configured in the same way as that PR... so it should be straight-forward to switch from the old branch to a new release containing the feature.

**Comment by mnlevy1981** [comment]
It turns out the `version-dropdown` branch used in the pull request linked above _is_ identical to the code I got from the Unidata group. PR #294 uses option 3a described in the first comment, which was easier to implement than option 1 because the `rtfd/sphinx_rtd_theme` master branch has some weird properties in `.gitattributes` and that led to `pip` compaining when I tried to install the branch on my laptop:

```
$ pip install git+https://github.com/rtfd/sphinx_rtd_theme.git@version-dropdown
Collecting git+https://github.com/rtfd/sphinx_rtd_theme.git@version-dropdown
  Cloning https://github.com/rtfd/sphinx_rtd_theme.git (to revision version-dropdown) to /private/var/folders/_l/75__d3ks5b54x_gklwjy2gc1_cwn99/T/pip-req-build-OyQT1b
error: Your local changes to the following files would be overwritten by checkout:
	docs/make.bat
Please commit your changes or stash them before you switch branches.
Aborting
Command "git checkout -q b46bba51649fd15b90e6938b57a1e96f7063c81f" failed with error code 1 in /private/var/folders/_l/75__d3ks5b54x_gklwjy2gc1_cwn99/T/pip-req-build-OyQT1b
```

There's an explicit `text eol=lf` that results in `docs/make.bat` being modified immediately after cloning, and then pip can't change from `master` to `version-dropdown` due to the changes in that file. So I forked [rtfd/sphinx_rtd_theme](https://github.com/rtfd/sphinx_rtd_theme) to [the marbl-ecosys org](https://github.com/marbl-ecosys/sphinx_rtd_theme) and made `version-dropdown` the default branch.

**Comment by mnlevy1981** [comment]
closed via #294
